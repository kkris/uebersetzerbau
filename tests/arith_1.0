minus = fun x -> fun y -> x - y end end;
