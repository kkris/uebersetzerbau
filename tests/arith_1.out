ident minus
=
fun
ident x
->
fun
ident y
->
ident x
-
ident y
end
end
;
